scripts/azureml-assets/azureml/assets/config.py

```diff
@@ -345,7 +344,94 @@ def __init__(self, branch: str, uri: str):
         self._branch = branch
         super().__init__(PathType.GIT, uri)
 
+class PathFactory():
+    """Factory method to create asset path."""
+    def __init__(self):
+        self.asset_path = None
+
+    def getPath(self, path, path_type)-> AssetPath:
+        if path_type == PathType.AZUREBLOB.value:
+            self.asset_path = AzureBlobstoreAssetPath(
+                storage_name=path['storage_name'],
+                container_name=path['container_name'],
+                container_path=path['container_path'],
+            )
+        elif path_type == PathType.GIT.value:
+            self.asset_path = GitAssetPath(branch=path['branch'], uri=path['uri'])
+        elif path_type == PathType.LOCAL.value:
+            self.asset_path= LocalAssetPath(local_path=path['uri'])
+        elif path_type == PathType.HTTP.value or path_type == PathType.FTP.value:
+            raise NotImplementedError("Support for HTTP and FTP is being added.")
+        return self.asset_path    

```

Comment:
Would you please try making this a `@classmethod` of `AssetPath` and renaming it to something like `from_dict`? Also make sure you use type indicators in the function args. I don't think we need to have a separate class for this.